tests/conftest.py

```diff
@@ -98,8 +98,14 @@ def app():
         SECURITY_DEPRECATED_PASSWORD_SCHEMES=[],
         OAUTHLIB_INSECURE_TRANSPORT=True,
         OAUTH2_CACHE_TYPE='simple',
+        RECORDS_UI_DEFAULT_PERMISSION_FACTORY='invenio_records_rest.utils.'
+        'allow_all'
     )
     app_.url_map.converters['pid'] = PIDConverter
+
+    # initialize InvenioDeposit first in order to detect any invalid dependency
+    InvenioDepositREST(app_)
```

Comment:
Why? It seems to me that [we are still registering InvenioDepositRest after all the other extensions](https://github.com/inveniosoftware/invenio-deposit/blob/1f112b67a4bdef0c810ece362ddc1af0184a9d94/tests/conftest.py#L132). Shouldn't we force as a best practice every module to load its extensions first just as a safeguard. If it was the case, the bug this PR fixes would have been detected right away.